site/src/components/LicenseCard/LicenseCard.tsx

```diff
@@ -0,0 +1,156 @@
+import Box from "@material-ui/core/Box"
+import Button from "@material-ui/core/Button"
+import Paper from "@material-ui/core/Paper"
+import { makeStyles } from "@material-ui/core/styles"
+import { License } from "api/typesGenerated"
+import { ConfirmDialog } from "components/Dialogs/ConfirmDialog/ConfirmDialog"
+import { Stack } from "components/Stack/Stack"
+import dayjs from "dayjs"
+import { useState } from "react"
+
+type LicenseCardProps = {
+  license: License
+  userLimitActual?: number
+  userLimitLimit?: number
+  onRemove: (licenseId: number) => void
+  isRemoving: boolean
+}
+
+export const LicenseCard = ({
+  license,
+  userLimitActual,
+  userLimitLimit,
+  onRemove,
+  isRemoving,
+}: LicenseCardProps) => {
+  const styles = useStyles()
+
+  const [licenseIDMarkedForRemoval, setLicenseIDMarkedForRemoval] = useState<
+    number | undefined
+  >(undefined)
+
+  return (
+    <Paper
+      variant="outlined"
+      key={license.id}
+      elevation={2}
+      className={styles.licenseCard}
+    >
+      <ConfirmDialog
+        type="info"
+        hideCancel={false}
+        open={licenseIDMarkedForRemoval !== undefined}
+        onConfirm={() => {
+          if (!licenseIDMarkedForRemoval) {
+            return
+          }
+          onRemove(licenseIDMarkedForRemoval)
+          setLicenseIDMarkedForRemoval(undefined)
+        }}
+        onClose={() => setLicenseIDMarkedForRemoval(undefined)}
+        title="Confirm license removal"
+        confirmLoading={isRemoving}
+        confirmText="Remove"
+        description="Are you sure you want to remove this license?"
+      />
+      <Stack
+        direction="column"
+        className={styles.cardContent}
+        justifyContent="space-between"
+      >
+        <Box className={styles.licenseId}>
+          <span>#{license.id}</span>
+        </Box>
+        <Stack className={styles.accountType}>
+          <span>{license.claims.trial ? "Trial" : "Enterprise"}</span>
+        </Stack>
```

Comment:
I think it might be nice to include the `license.uploaded_at` value somewhere on this card. Currently, we don't have a way to name licenses (this would be a helpful feature addition down the road), so it's a little hard to know which license is which, especially when deleting. `license.uploaded_at` might help orient the user so they can delete with more confidence.

Reply:
I will implement this in a second PR. We discussed this a while ago and most of our users only have one License.  So I believe we can leave this for a second iteration